src/components/Banner/Banner.stories.tsx

```diff
@@ -1,156 +1,219 @@
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
-      </p>
-    </TextPassage>
-  </Banner>
-);
-
-const WithActionTemplate: Story<Props> = (args) => (
-  <Banner {...args}>
-    <TextPassage className="u-margin-bottom-md">
-      <p>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
-      </p>
-    </TextPassage>
-    <Button>Banner action</Button>
-  </Banner>
-);
-
-export const Default = Template.bind({});
-Default.args = { iconName: 'info' };
-
-export const Brand = Template.bind({});
-Brand.args = {
-  variant: 'brand',
-  iconName: 'check-circle',
-  title: 'This is a title',
-};
-
-export const Success = Template.bind({});
-Success.args = {
-  variant: 'success',
-  iconName: 'check-circle',
-  title: 'This is a title',
-};
-
-export const Warning = Template.bind({});
-Warning.args = {
-  variant: 'warning',
-  iconName: 'error',
-  title: 'This is a title',
-};
-
-export const Error = Template.bind({});
-Error.args = {
-  variant: 'error',
-  iconName: 'cancel',
-  title: 'This is a title',
-};
-
-export const Dismissible = Template.bind({});
-Dismissible.args = {
-  dismissible: true,
-  iconName: 'info',
-  title: 'This is a title',
-};
-
-export const BrandDismissible = Template.bind({});
-BrandDismissible.args = {
-  dismissible: true,
-  variant: 'brand',
-  iconName: 'check-circle',
-  title: 'This is a title',
-};
-
-export const SuccessDismissible = Template.bind({});
-SuccessDismissible.args = {
-  dismissible: true,
-  variant: 'success',
-  iconName: 'check-circle',
-  title: 'This is a title',
-};
-
-export const WarningDismissible = Template.bind({});
-WarningDismissible.args = {
-  dismissible: true,
-  variant: 'warning',
-  iconName: 'error',
-  title: 'This is a title',
-};
-
-export const ErrorDismissible = Template.bind({});
-ErrorDismissible.args = {
-  dismissible: true,
-  variant: 'error',
-  iconName: 'cancel',
-  title: 'This is a title',
-};
-
-export const TitleOnly = TitleOnlyTemplate.bind({});
-TitleOnly.args = {
-  dismissible: true,
-  iconName: 'info',
-  title: 'This is a title',
-};
-
-export const DescriptionOnly = DescriptionOnlyTemplate.bind({});
-DescriptionOnly.args = {
-  dismissible: true,
-  iconName: 'info',
-  title: 'This is a title',
-};
-
-export const WithAction = WithActionTemplate.bind({});
-WithAction.args = {
-  dismissible: true,
-  iconName: 'info',
-  title: 'This is a title',
+  args: {
+    content: (
+      <>
+        Summit Learning has a full-time team dedicated to constantly improving
+        our curriculum. To see the updates,{' '}
+        <Button
+          href="/"
+          onClick={(event) => event.preventDefault()}
+          variant="link"
+        >
+          click into the course
+        </Button>
+        .
+      </>
+    ),
+    heading:
```

Comment:
suuper nit bc I got confused 😅  is there a reason you didn't name these `description` & `title`?

Reply:
Nah, this was just leftover from before. Updating, thanks!